<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchBoxDialog</class>
 <widget class="QDialog" name="SearchBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="14,0,68">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="347,35">
     <item alignment="Qt::AlignHCenter">
      <widget class="CustomLabel" name="titleLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomPushButton" name="closePushButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="CustomLabel" name="searchTextLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1" columnstretch="70,154,56,154,50,70,54">
       <item row="1" column="5">
        <widget class="CustomCheckBox" name="checkBox_2">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="CustomCheckBox" name="checkBox">
         <property name="text">
          <string>1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="CustomPushButton" name="secondPlusButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="CustomPushButton" name="secondValueButton">
         <property name="text">
          <string>60</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="CustomPushButton" name="searchButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="CustomPushButton" name="firstMinusButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="CustomPushButton" name="firstPlusButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3">
        <widget class="CustomPushButton" name="secondMinusButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="CustomPushButton" name="firstValueButton">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPushButton</class>
   <extends>QPushButton</extends>
   <header>CustomPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomCheckBox</class>
   <extends>QCheckBox</extends>
   <header>CustomCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
